reana_job_controller/cli.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""Click command-line interface for REANA Job Controller."""
+
+import json
+
+import click
+from flask import current_app
+from flask.cli import with_appcontext
+
+from .spec import build_openapi_spec
+
+
+@click.group()
+def openapi():
+    """Openapi management commands."""
+
+
+@openapi.command()
+@click.argument('output', type=click.File('w'))
+@with_appcontext
+def create(output):
```

Comment:
It would be good to plug CLI call into Sphinx documentation.  Perhaps a new "CLI" chapter, perhaps simply under "Getting started" for now.